include/nuttx/can/can.h

```diff
@@ -246,6 +257,46 @@
  *                   is returned with the errno variable set to indicate the
  *                   nature of the error.
  *   Dependencies:   None
+ *
+ * CANIOC_SET_STATE
```

Comment:
How will this work? If I pass `CAN_STATE_STOP` via the ioctl then I stop the controller? What will happen with currently queued frames? Will the frame currently being transferred aborted or will the ioctlc all wait for the driver to finish? I generally think it is better to have a separate call to start or stop the controller. And then an ioctl to obtain current controller's state (including error active, passive, etc.).